docs/docs/exosphere/retry-policy.md

```
@@ -0,0 +1,395 @@
+
+The `_cap` function is applied to all calculated delays:
+
+```python
+def _cap(value: int) -> int:
+    if self.max_delay is not None:
+        return min(value, self.max_delay)
+    return value
+```
+
+**Behavior:**
+- If `max_delay` is set, all calculated delays are capped at this value
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)